template breaking on transparent color inline replacement

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I am currently playing around with the transient prompt. Sadly not as easy as with the segments to build something nice and reduced, but I am making progress.

I noticed an interesting bug though. The template I am basing mine off of **night-owl**, used `"transparent"` as the background color for the transient prompt.  
No issues if you keep it as is. But if you modify the `"template"` for it in any way and add [Color Overrides](https://ohmyposh.dev/docs/configuration/colors#color-overrides) inside, it breaks.

From what I tested, it's **very** specific to the actual string "transparent" or something. Likely because you use some specific replacement for it to get ARGB, but then the index is messed up?

So, it looks like this when it is broken:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/9962104/adb392f3-ddbf-4a08-bf6f-c60c2095f136)

And like this if I set the color to `"black"` instead:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/9962104/a6695b55-302c-4bf3-a3e6-f35391613380)

You can see how it correctly switched back the font color, but it added `ransparentm` instead of replacing the background color 🙃 

Here is how I have my transient prompt, to preproduce. Though any scenario should make this break.
```json
  "transient_prompt": {
    "background": "transparent",
    "foreground": "#d6deeb",
    "template": "\u256d\u2500\ue0b6 <#011627,#82AAFF> {{ path .Segments.Path.Path .Segments.Path.Location }} </> \ue285 "
  },
```

And like i said, it works if you set it to `"background": "black"`.  
Funnily enough, it **also** works as expected if you had the original background as something invalid. I tried `"background": "blubb"` and it *worked*.

I feel like this might also happen in other template strings, when the background of the element itself is set to transparent? Haven't tested though.

Made an empty command line for the log, just hitting enter in root folder:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/9962104/578fd3af-e111-4677-8ab6-2663d866fb98)


### Theme

Just a slightly modified version of `night-owl.omp.json` at the moment.

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Well, if that helps...


PowerShell 7.4.2
╭─     ransparentm                                                                                                                       ╭─     ransparentm  oh-my-posh debug --plain

Version: 20.2.2

Shell: pwsh (7.4.2)

Prompt:

╭─      0ms                                                                                             pwsh  38%  04:19:27
╰─  Five Finger Death Punch ~ Bad Company  

Segments:

ConsoleTitle(true)                         -   0 ms
Os(true)                                   -   0 ms
Root(false)                                -   0 ms
Path(true)                                 -   0 ms
Git(false)                                 -   5 ms
Executiontime(true)                        -   0 ms
Shell(true)                                -   0 ms
Sysinfo(true)                              -   0 ms
Angular(false)                             -   0 ms
Aws(false)                                 -   0 ms
Az(false)                                  -   0 ms
Azfunc(false)                              -   0 ms
Cds(false)                                 -   0 ms
Crystal(false)                             -   0 ms
Cf(false)                                  -   0 ms
Cftarget(false)                            -   9 ms
Cmake(false)                               -   0 ms
Dart(false)                                -   0 ms
Dotnet(false)                              -   0 ms
Flutter(false)                             -   0 ms
Go(false)                                  -   0 ms
Haskell(false)                             -   0 ms
Java(false)                                -   0 ms
Julia(false)                               -   0 ms
Kotlin(false)                              -   0 ms
Kubectl(false)                             -   9 ms
Lua(false)                                 -   0 ms
Node(false)                                -   0 ms
Nx(false)                                  -   0 ms
Perl(false)                                -   0 ms
Php(false)                                 -   0 ms
Python(false)                              -   0 ms
R(false)                                   -   0 ms
Ruby(false)                                -   0 ms
Rust(false)                                -   0 ms
Swift(false)                               -   0 ms
Time(true)                                 -   0 ms
Text(true)                                 -   0 ms
Spotify(true)                              -   2 ms
Status(true)                               -   0 ms

Run duration: 29.9072ms

Cache path: C:\Users\Wolfsblvt\AppData\Local\oh-my-posh

Config path: C:\Users\Wolfsblvt\.wolf_nightowl.omp.json

Logs:

[DEBUG] 04:19:27.427 shell.go:Getenv:326 → NO DATA
[TRACE] 04:19:27.427 shell.go:Getenv(TERM_PROGRAM) - 508.9µs
[DEBUG] 04:19:27.427 shell.go:Getenv:326 → C:\Users\Wolfsblvt\AppData\Local
[TRACE] 04:19:27.427 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 04:19:27.427 shell_windows.go:CachePath() - 0s
[DEBUG] 04:19:27.428 shell.go:Getenv:326 → C:\Users\Wolfsblvt\.wolf_nightowl.omp.json
[TRACE] 04:19:27.428 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 04:19:27.428 shell.go:Shell:601 → no shell name provided in flags, trying to detect it
[DEBUG] 04:19:27.436 shell.go:Shell:609 → process name: pwsh.exe
[TRACE] 04:19:27.436 shell.go:Shell() - 7.739ms
[TRACE] 04:19:27.436 shell.go:resolveConfigPath() - 7.739ms
[TRACE] 04:19:27.436 shell.go:Init() - 9.2615ms
[TRACE] 04:19:27.436 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 config.go:loadConfig() - 1.6348ms
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[DEBUG] 04:19:27.438 shell.go:Getenv:326 → NO DATA
[TRACE] 04:19:27.438 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 04:19:27.438 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 shell_windows.go:WindowsRegistryKeyValue:217 → ColorizationColor(DWORD): 0xC4234A52
[TRACE] 04:19:27.438 shell.go:Shell() - 0s
[DEBUG] 04:19:27.438 shell.go:Getenv:326 → 7.4.2
[TRACE] 04:19:27.438 shell.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 04:19:27.438 debug.go:PrintDebug:22 → Segment: Title
[DEBUG] 04:19:27.438 text.go:Render:71 → Rendering template: {{ .Folder }}
[TRACE] 04:19:27.438 shell_windows.go:Root() - 502.3µs
[TRACE] 04:19:27.438 shell.go:Shell() - 0s
[TRACE] 04:19:27.438 shell.go:StatusCodes() - 0s
[TRACE] 04:19:27.438 shell_windows.go:IsWsl() - 0s
[DEBUG] 04:19:27.438 shell.go:TemplateCache:782 → environment: [ACSetupSvcPort=23210 ACSvcPort=17532 ALLUSERSPROFILE=C:\ProgramData APPDATA=C:\Users\Wolfsblvt\AppData\Roaming ChocolateyInstall=C:\ProgramData\chocolatey ChocolateyLastPathUpdate=133508036630865601 CommonProgramFiles=C:\Program Files\Common Files CommonProgramFiles(x86)=C:\Program Files (x86)\Common Files CommonProgramW6432=C:\Program Files\Common Files COMPUTERNAME=GERALT ComSpec=C:\Windows\system32\cmd.exe CONDA_PROMPT_MODIFIER=False DriverData=C:\Windows\System32\Drivers\DriverData HOMEDRIVE=C: HOMEPATH=\Users\Wolfsblvt LOCALAPPDATA=C:\Users\Wolfsblvt\AppData\Local LOGONSERVER=\\GERALT NUMBER_OF_PROCESSORS=16 NVM_HOME=C:\Users\Wolfsblvt\AppData\Roaming\nvm NVM_SYMLINK=C:\Program Files\nodejs OneDrive=D:\OneDrive OneDriveConsumer=D:\OneDrive OS=Windows_NT Path=C:\Program Files\WindowsApps\Microsoft.PowerShell_7.4.2.0_x64__8wekyb3d8bbwe;C:\Python312\Scripts\;C:\Python312\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\dotnet\;%NVM_HOME%;%NVM_SYMLINK%;C:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files\Git\cmd;C:\Program Files\ExifTool;C:\Users\Wolfsblvt\AppData\Local\Programs\Python\Python310\Scripts\;C:\Users\Wolfsblvt\AppData\Local\Programs\Python\Python310\;C:\Users\Wolfsblvt\AppData\Local\Microsoft\WindowsApps;C:\Users\Wolfsblvt\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\Wolfsblvt\AppData\Roaming\nvm;C:\Program Files\nodejs;C:\Users\Wolfsblvt\AppData\Local\GitHubDesktop\bin;C:\Users\Wolfsblvt\AppData\Roaming\npm;C:\Users\Wolfsblvt\AppData\Local\Programs\oh-my-posh\bin PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW;.CPL POSH_AZURE_ENABLED=False POSH_CURSOR_COLUMN=1 POSH_CURSOR_LINE=3 POSH_GIT_ENABLED=False POSH_INSTALLER=winget POSH_PID=36844 POSH_SHELL_VERSION=7.4.2 POSH_THEME=C:\Users\Wolfsblvt\.wolf_nightowl.omp.json POSH_THEMES_PATH=C:\Users\Wolfsblvt\AppData\Local\Programs\oh-my-posh\themes POWERLINE_COMMAND=oh-my-posh PROCESSOR_ARCHITECTURE=AMD64 PROCESSOR_IDENTIFIER=Intel64 Family 6 Model 158 Stepping 13, GenuineIntel PROCESSOR_LEVEL=6 PROCESSOR_REVISION=9e0d ProgramData=C:\ProgramData ProgramFiles=C:\Program Files ProgramFiles(x86)=C:\Program Files (x86) ProgramW6432=C:\Program Files PSModulePath=C:\Users\Wolfsblvt\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\windowsapps\microsoft.powershell_7.4.2.0_x64__8wekyb3d8bbwe\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules PUBLIC=C:\Users\Public RlsSvcPort=22112 SESSIONNAME=Console SystemDrive=C: SystemRoot=C:\Windows TEMP=C:\Users\WOLFSB~1\AppData\Local\Temp TMP=C:\Users\WOLFSB~1\AppData\Local\Temp USERDOMAIN=GERALT USERDOMAIN_ROAMINGPROFILE=GERALT USERNAME=Wolfsblvt USERPROFILE=C:\Users\Wolfsblvt windir=C:\Windows WSLENV=WT_SESSION:WT_PROFILE_ID: WT_PROFILE_ID={574e775e-4f2a-5b96-ac1e-a2962a402336} WT_SESSION=47fe81ba-d951-4b43-ad5f-1e2061fa5dac]
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 shell.go:Pwd:356 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.438 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.438 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 shell.go:User:494 → Wolfsblvt
[TRACE] 04:19:27.438 shell.go:User() - 0s
[DEBUG] 04:19:27.438 shell.go:Host:511 → Geralt
[TRACE] 04:19:27.438 shell.go:Host() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 shell.go:Getenv:326 → NO DATA
[TRACE] 04:19:27.438 shell.go:Getenv(SHLVL) - 0s
[TRACE] 04:19:27.438 shell.go:TemplateCache() - 502.3µs
[DEBUG] 04:19:27.438 shell.go:Getenv:326 → 3
[TRACE] 04:19:27.438 shell.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 04:19:27.438 shell.go:Getenv:326 → 1
[TRACE] 04:19:27.438 shell.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 shell.go:Shell() - 0s
[TRACE] 04:19:27.438 shell.go:Shell() - 0s
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 shell.go:Pwd() - 0s
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.438 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.438 shell.go:Pwd() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:Pwd() - 0s
[TRACE] 04:19:27.438 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.438 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.438 segment.go:SetEnabled:533 → Segment: Executiontime
[DEBUG] 04:19:27.438 segment.go:SetEnabled:533 → Segment: Root
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 segment.go:SetEnabled:533 → Segment: Path
[DEBUG] 04:19:27.438 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.438 segment.go:SetEnabled:533 → Segment: Git
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 segment.go:SetEnabled:533 → Segment: Os
[TRACE] 04:19:27.438 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.438 properties.go:GetBool:22 → always_enabled: false
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:Shell() - 0s
[TRACE] 04:19:27.438 shell.go:ExecutionTime() - 0s
[TRACE] 04:19:27.438 shell.go:Flags() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 properties.go:GetString:28 → 
[TRACE] 04:19:27.438 shell_windows.go:Root() - 0s
[TRACE] 04:19:27.438 shell.go:TemplateCache() - 0s
[DEBUG] 04:19:27.438 properties.go:GetString:28 → roundrock
[DEBUG] 04:19:27.438 shell.go:Getenv:326 → C:\Users\Wolfsblvt\AppData\Local
[TRACE] 04:19:27.438 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.438 shell.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 04:19:27.438 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 04:19:27.438 properties.go:GetString:28 → 
[DEBUG] 04:19:27.438 properties.go:GetString:28 → 
[DEBUG] 04:19:27.438 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 properties.go:GetString:28 →  
[DEBUG] 04:19:27.438 properties.go:GetKeyValueMap:46 → mapped_locations: map[C:\Program Files: 󰣆 󰬿󰬽  C:\Program Files (x86): 󰣆 󰬼󰬻  C:\Windows: 󰨡  D:\OneDrive:  ]
[DEBUG] 04:19:27.438 text.go:Render:71 → Rendering template: C:\Program Files
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 text.go:Render:71 → Rendering template: C:\Program Files (x86)
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 text.go:Render:71 → Rendering template: C:\Windows
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.438 text.go:Render:71 → Rendering template: D:\OneDrive
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[DEBUG] 04:19:27.438 properties.go:GetString:28 → NO DATA
[TRACE] 04:19:27.438 shell.go:GOOS() - 0s
[TRACE] 04:19:27.438 shell.go:PathSeparator() - 0s
[TRACE] 04:19:27.438 shell.go:Pwd() - 0s
[TRACE] 04:19:27.438 shell_windows.go:IsWsl() - 0s
[TRACE] 04:19:27.438 shell.go:StackCount() - 0s
[DEBUG] 04:19:27.439 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 04:19:27.439 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 04:19:27.439 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 04:19:27.439 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 04:19:27.439 shell_windows.go:DirIsWritable() - 510.8µs
[TRACE] 04:19:27.439 shell.go:TemplateCache() - 0s
[DEBUG] 04:19:27.443 shell.go:CommandPath:553 → C:\Program Files\Git\cmd\git.exe
[TRACE] 04:19:27.443 shell.go:CommandPath(git.exe) - 5.1085ms
[TRACE] 04:19:27.443 shell.go:HasCommand(git.exe) - 5.1085ms
[TRACE] 04:19:27.443 shell.go:Pwd() - 0s
[ERROR] 04:19:27.443 shell.go:HasParentFilePath:697 → CreateFile .git: The system cannot find the file specified.
[TRACE] 04:19:27.443 shell.go:HasParentFilePath(.git) - 0s
[DEBUG] 04:19:27.443 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 04:19:27.443 text.go:Render:71 → Rendering template:  {{ .FormattedMs }}
[DEBUG] 04:19:27.443 text.go:Render:71 → Rendering template:  {{ if .WSL }}WSL at {{ end }}{{.Icon}}
[DEBUG] 04:19:27.443 text.go:Render:71 → Rendering template: {{ path .Path .Location }}
[TRACE] 04:19:27.444 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.444 shell.go:Shell() - 0s
[TRACE] 04:19:27.444 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.444 shell.go:Shell() - 0s
[TRACE] 04:19:27.444 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.444 shell.go:Shell() - 0s
[DEBUG] 04:19:27.444 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.444 text.go:Render:71 → Rendering template: powerline
[DEBUG] 04:19:27.444 text.go:Render:71 → Rendering template: powerline
[DEBUG] 04:19:27.444 text.go:Render:71 → Rendering template: powerline
[DEBUG] 04:19:27.444 text.go:Render:71 → Rendering template: diamond
[TRACE] 04:19:27.444 shell.go:Shell() - 0s
[TRACE] 04:19:27.444 shell.go:Shell() - 0s
[TRACE] 04:19:27.444 shell.go:Shell() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 properties.go:GetInt:40 → precision: 2
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Az
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Angular
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 properties.go:GetString:28 → first_match
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Shell
[DEBUG] 04:19:27.444 properties.go:GetString:28 → context
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 shell.go:Getenv:326 → NO DATA
[DEBUG] 04:19:27.444 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 04:19:27.444 shell.go:Getenv(AZURE_CONFIG_DIR) - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[TRACE] 04:19:27.444 shell.go:Shell() - 0s
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [angular.json]
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Cftarget
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Aws
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.444 shell.go:TemplateCache() - 0s
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.444 shell.go:Getenv:326 → C:\Users\Wolfsblvt\AppData\Local
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Sysinfo
[DEBUG] 04:19:27.444 shell.go:Getenv:326 → NO DATA
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.444 shell.go:Getenv(AWS_DEFAULT_REGION) - 0s
[ERROR] 04:19:27.444 shell.go:HasFilesInDir:378 → open .: The system cannot find the file specified.
[DEBUG] 04:19:27.444 properties.go:GetString:28 → environment
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [*.cs *.csx *.vb *.sln *.slnf *.csproj *.vbproj *.fs *.fsx *.fsproj global.json]
[DEBUG] 04:19:27.444 shell.go:Getenv:326 → NO DATA
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.444 shell.go:HasFilesInDir:379 → false
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:Getenv(AWS_CONFIG_FILE) - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.444 shell.go:HasFilesInDir(azureProfile.json) - 0s
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Python
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Flutter
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [*.dart pubspec.yaml pubspec.yml pubspec.lock]
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Time
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → folders: [.dart_tool]
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 properties.go:GetString:28 → 15:04:05
[TRAUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Crystal
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.444 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Cmake
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [*.cmake CMakeLists.txt]
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [*.cr shard.yml]
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → folders: []
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → folders: []
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Dart
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Nx
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Go
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [workspace.json nx.json]
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [*.go go.mod]
[DEBUG] 04:19:27.444 prope.go:Pwd() - 0s
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [*.dart pubspec.yaml pubspec.yml pubspec.lock]
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 properties.go:GetBool:22 → home_enabled: false
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[ERROR] 04:19:27.444 shell.go:HasFilesInDir:378 → open .: The system cannot find the file specified.
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → folders: [.dart_tool]
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 shell.go:Getenv:326 → NO DATA
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Getenv(POSH_AZURE_SUBSCRIPTION) - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Rust
[TRACE] 04:19:27.444 shell.go:Pwd() - 0s
[TRACE] 04:19:27.444 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.444 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.444 properties.go:GetStringArray:52 → extensions: [*.rs Cargo.toml Cargo.lock]
[DEBUG] 04:19:27.444 segment.go:SetEnabled:533 → Segment: Kotlin
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [*.kt *.kts *.ktm]
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [*.jl]
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[TRACE] 04:19:27.445 shell.go:Flags() - 0s
[TRACE] 04:19:27.444 shell.go:Flags() - 0s
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.445 shell.go:Flags() - 0s
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.445 properties.go:GetString:28 → lua
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Perl
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [.perl-version *.pl *.pm *.t]
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[TRACE] 04:19:27.445 shell.go:Flags() - 0s
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Lua
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [*.lua *.rockspec]
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: [lua]
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [*.R *.Rmd *.Rsx *.Rda *.Rd *.Rproj .Rproj.user]
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.445 properties.go:GetString:28 → never
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[TRACE] 04:19:27.445 shell.go:Flags() - 0s
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Ruby
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.445 properties.go:GetString:28 → files
[TRACE] 04:19:27.445 shell.go:Flags() - 0s
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Haskell
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [*.rb Rakefile Gemfile]
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 shell.go:Getenv:326 → NO DATA
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [*.hs *.lhs stack.yaml package.yaml *.cabal cabal.project]
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[TRACE] 04:19:27.445 shell.go:Getenv(JAVA_HOME) - 0s
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Swift
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [*.swift *.SWIFT]
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Kubectl
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Cf
[TRACE] 04:19:27.445 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Php
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → parse_kubeconfig: false
[DEBUG] 04:19:27.445 segment.go:SetEnabled:533 → Segment: Java
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [*.php composer.json composer.lock .php-version blade.php]
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [manifest.yml mta.yaml]
[DEBUG] 04:19:27.445 shell.go:Getenv:326 → C:\Users\Wolfsblvt\AppData\Local
[TRACE] 04:19:27.445 shell.go:Getenv(LOCALAPPDATA) - 0s
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → extensions: [pom.xml build.gradle.kts build.sbt .java-version .deps.edn project.clj build.boot *.java *.class *.gradle *.jar *.clj *.cljc]
[DEBUG] 04:19:27.445 properties.go:GetStringArray:52 → folders: []
[DEBUG] 04:19:27.445 properties.go:GetBool:22 → home_enabled: false
[TRACE] 04:19:27.445 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.445 shell_windows.go:58 → C:\Users\Wolfsblvt
[ERROR] 04:19:27.454 shell.go:CommandPath:557 → The system cannot find the file specified.
[TRACE] 04:19:27.454 shell.go:CommandPath(cf) - 9.7132ms
[TRACE] 04:19:27.454 shell.go:HasCommand(cf) - 9.7132ms
[ERROR] 04:19:27.455 shell.go:CommandPath:557 → The system cannot find the file specified.
[TRACE] 04:19:27.455 shell.go:CommandPath(kubectl) - 9.7037ms
[TRACE] 04:19:27.455 shell.go:HasCommand(kubectl) - 9.7037ms
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template:   {{ .Name }}
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template:  {{ round .PhysicalPercentUsed .Precision }}%
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template:  {{ .CurrentDate | date .Format }}
[TRACE] 04:19:27.455 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.455 shell.go:Shell() - 0s
[TRACE] 04:19:27.455 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.455 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.455 shell.go:Shell() - 0s
[TRACE] 04:19:27.455 shell.go:Shell() - 0s
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.455 shell_windows.go:TerminalWidth:111 → terminal width: 140
[TRACE] 04:19:27.455 shell_windows.go:TerminalWidth() - 0s
[TRACE] 04:19:27.455 shell.go:Shell() - 0s
[TRACE] 04:19:27.455 shell.go:Shell() - 0s
[TRACE] 04:19:27.455 shell.go:Shell() - 0s
[TRACE] 04:19:27.455 shell.go:Flags() - 0s
[TRACE] 04:19:27.455 shell.go:Flags() - 0s
[TRACE] 04:19:27.455 shell.go:Flags() - 0s
[TRACE] 04:19:27.455 shell.go:Pwd() - 0s
[TRACE] 04:19:27.455 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.455 properties.go:GetString:28 → {{ .Code }}
[TRACE] 04:19:27.455 shell.go:Pwd() - 0s
[DEBUG] 04:19:27.455 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.455 shell_windows.go:58 → C:\Users\Wolfsblvt
[TRACE] 04:19:27.455 shell.go:GOOS() - 0s
[TRACE] 04:19:27.455 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.455 shell_windows.go:58 → C:\Users\Wolfsblvt
[DEBUG] 04:19:27.455 segment.go:SetEnabled:533 → Segment: Text
[TRACE] 04:19:27.455 shell.go:GOOS() - 0s
[DEBUG] 04:19:27.455 segment.go:SetEnabled:533 → Segment: Spotify
[DEBUG] 04:19:27.455 segment.go:SetEnabled:533 → Segment: Status
[TRACE] 04:19:27.455 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.455 shell.go:StatusCodes() - 0s
[DEBUG] 04:19:27.455 text.go:Render:71 → Rendering template: {{ .Code }}
[TRACE] 04:19:27.455 shell.go:TemplateCache() - 0s
[DEBUG] 04:19:27.455 properties.go:GetBool:22 → always_enabled: true
[TRACE] 04:19:27.455 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.457 shell_windows.go:QueryWindowTitles(^(Spotify.*)|(.*\s-\s.*)$) - 2.009ms
[DEBUG] 04:19:27.457 properties.go:GetString:28 → 
[TRACE] 04:19:27.457 shell.go:TemplateCache() - 0s
[DEBUG] 04:19:27.457 text.go:Render:71 → Rendering template: 
[TRACE] 04:19:27.457 shell.go:Shell() - 0s
[DEBUG] 04:19:27.457 text.go:Render:71 → Rendering template: ╰─
[DEBUG] 04:19:27.457 text.go:Render:71 → Rendering template:  {{ .Icon }}{{ if ne .Status "stopped" }}{{ .Artist }} ~ {{ .Track }}{{ end }}
[TRACE] 04:19:27.457 shell.go:Shell() - 0s
[TRACE] 04:19:27.457 shell.go:TemplateCache() - 0s
[TRACE] 04:19:27.457 shell.go:Shell() - 0s
[DEBUG] 04:19:27.457 shell.go:Getenv:326 → NO DATA
[TRACE] 04:19:27.457 shell.go:Getenv(TERM_PROGRAM) - 0s
[DEBUG] 04:19:27.457 text.go:Render:71 → Rendering template: plain
[DEBUG] 04:19:27.457 text.go:Render:71 → Rendering template: diamond
[DEBUG] 04:19:27.457 text.go:Render:71 → Rendering template: {{ if gt .Code 0 }}#ef5350{{ end }}
[TRACE] 04:19:27.457 shell.go:TemplateCache() - 0s
[DEBUG] 04:19:27.457 text.go:Render:71 → Rendering template: plain
[TRACE] 04:19:27.457 shell.go:Shell() - 0s
[TRACE] 04:19:27.457 shell.go:Flags() - 0s
[TRACE] 04:19:27.457 shell.go:Shell() - 0s
[TRACE] 04:19:27.457 shell.go:Shell() - 0s
[DEBUG] 04:19:27.457 shell.go:Getenv:326 → C:\Users\Wolfsblvt\AppData\Local
[TRACE] 04:19:27.457 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 04:19:27.457 shell_windows.go:CachePath() - 0s
[TRACE] 04:19:27.457 shell.go:Flags() - 0s
╭─      70ms                                                                                            pwsh  38%  04:19:27
╰─  Five Finger Death Punch ~ Bad Company  
```
```

--- a/src/ansi/ansi_writer.go
+++ b/src/ansi/ansi_writer.go
@@ -558,7 +558,12 @@ func (w *Writer) endColorOverride(position int) int {
 		}
 
 		if previousBg != bg {
-			w.writeEscapedAnsiString(fmt.Sprintf(colorise, previousBg))
+			background := fmt.Sprintf(colorise, previousBg)
+			if previousBg.IsClear() {
+				background = backgroundStyle.End
+			}
+
+			w.writeEscapedAnsiString(background)
 		}
 
 		if previousFg != fg {

--- a/src/ansi/ansi_writer_test.go
+++ b/src/ansi/ansi_writer_test.go
@@ -212,6 +212,12 @@ func TestWriteANSIColors(t *testing.T) {
 			Expected: "\x1b[47m\x1b[31m<\x1b[30m>\x1b[33m<\x1b[0m",
 			Colors:   &Colors{Foreground: "black", Background: "white"},
 		},
+		{
+			Case:     "Transparent override with parent",
+			Input:    "hello <#011627,#82AAFF>new</> world",
+			Expected: "\x1b[33mhello \x1b[48;2;130;170;255m\x1b[38;2;1;22;39mnew\x1b[49m\x1b[33m world\x1b[0m",
+			Colors:   &Colors{Foreground: "yellow", Background: "transparent"},
+		},
 	}
 
 	for _, tc := range cases {

